src/autogluon/bench/eval/scripts/run_evaluation_openml.py

```diff
@@ -150,15 +250,16 @@ def run(
         banned_datasets=banned_datasets,
         infer_batch_size=infer_batch_size,
         treat_folds_as_datasets=treat_folds_as_datasets,
+        clean_data=clean_data,
     )
 
     folds_to_keep = sorted(results_raw["fold"].unique())
 
-    if len(folds_to_keep) > 1:
+    if len(frameworks_run) > 1:
         compute_win_rate_per_dataset(
             f1=frameworks_run[0], f2=frameworks_run[1], results_raw=results_raw, folds=folds_to_keep
         )
-    if compute_z_score and len(folds_to_keep) > 1:
+    if compute_z_score and len(frameworks_run) > 1:
```

Comment:
`if compute_z_score and len(folds_to_keep) > 1 and len(frameworks_run) > 1:`

Reply:
Good catch, will update!